### PR TITLE
add extra packages to recent R easyconfigs

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
@boegelbot please test @ generoso
EB_ARGS="--skip"

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/9bce9b3496c190c812abeafc003ed887 for a full test report.

File: easybuild/easyconfigs/r/R/R-4.2.1-foss-2022a.eb [hunk 1]
```diff
@@ -3535,6 +3535,18 @@ exts_list = [
     ('svglite', '2.1.1', {
         'checksums': ['48700169eec1b05dbee9e2bae000aa84c544617b018cb3ac431a128cfd8dac56'],
     }),
+    ('ECOSolveR', '0.5.5', {
+        'checksums': ['2594ed1602b2fe159cc9aff3475e9cba7c1927b496c3daeabc1c0d227943ecc7'],
+    }),
+    ('scs', '3.2.4', {
+        'checksums': ['c3f39874bf4532fa8c2f2e2c41533ba4fe20b61cf6dfc6314407dc981621298f'],
+    }),
+    ('osqp', '0.6.0.8', {
```
Comment (branfosj) [inline easybuild/easyconfigs/r/R/R-4.2.1-foss-2022a.eb:3544]: https://github.com/osqp/osqp-r/blob/master/configure#L27

If a system `cmake` is found then it is used, if not it uses a custom `Makefile` to build `osqp`. They'll be a test fail report from me due to the OS `cmake` failing (`XZ_5.2` symbol issue, see #14991 and #5518).

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/80b480c5520a90386b5089cd96473e10 for a full test report.

File: easybuild/easyconfigs/r/R/R-4.2.1-foss-2022a.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/r/R/R-4.2.1-foss-2022a.eb:3544]: ok, i'll add CMake builddep

**Comment by smoors** [comment]
@boegelbot please test @ generoso EB_ARGS="--skip"

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2 EB_ARGS="--skip"

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/95dc89c341c41cc28b2f49d96e34caeb for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @smoors!

**Pull request merged**